cinema/serializers.py

```diff
@@ -36,6 +36,12 @@ class Meta:
         fields = ("id", "title", "description", "duration", "genres", "actors")
```

Comment:
Make following changes. Apply the same change to `MovieDetailSerializer` in case needed
```suggestion
        fields = ("id", "title", "description", "duration", "genres", "actors", "image")
        extra_kwargs = {"image": {"read_only": True}}
```